commerce-price-list-web/src/main/java/com/liferay/commerce/price/list/web/internal/display/context/BaseCommercePriceListDisplayContext.java

```diff
@@ -210,6 +210,7 @@ protected String getNavigation() {
 	protected final HttpServletRequest httpServletRequest;
 	protected final LiferayPortletRequest liferayPortletRequest;
 	protected final LiferayPortletResponse liferayPortletResponse;
+	protected CommercePriceListRequestHelper requestHelper;
```

Comment:
`CommercePriceListRequestHelper requestHelper` should be `CommercePriceListRequestHelper commercePriceListRequestHelper`